src/utils/multiline-expressions.ts

```diff
@@ -3,6 +3,11 @@ import * as vsc from 'vscode'
 export const AllTabs = /^\t+$/
 export const AllSpaces = /^ +$/
 
+export function simpleStripIndent(code: string, joinSep = '\n') {
+  const reNewLine = /\r?\n/
+  return code.split(reNewLine).map((line) => line.replace(/^\s*/, '')).join(joinSep)
+}

```

Comment:
What;s the benefit of that over simple `.replace(/\s+/g, ' ')`?

Reply:
Yeah, I like it

Reply:
Maybe also rename to `stripExcessiveWhitespace`?